src/sentry/api/analytics.py

```diff
@@ -34,6 +34,19 @@ class FunctionTimerEvent(analytics.Event):
     )
 
 
+class GroupSimilarIssuesEmbeddingsCountEvent(analytics.Event):
+    type = "group_similar_issues_embeddings.count"
+
+    attributes = (
+        analytics.Attribute("org_id"),
```

Comment:
not sure why this file has so many inconsistent, would have to update the code too
```suggestion
        analytics.Attribute("organization_id"),
```

Reply:
ugh created is org_id, just leave it